waku/v2/protocol/rln/metrics.go

```diff
@@ -0,0 +1,171 @@
+package rln
+
+import (
+	"time"
+
+	"github.com/libp2p/go-libp2p/p2p/metricshelper"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var messagesTotal = prometheus.NewCounter(
+	prometheus.CounterOpts{
+		Name: "waku_rln_messages_total",
+		Help: "number of messages received in the RLN validator",
+	})
+
+var spamMessagesTotal = prometheus.NewCounter(
+	prometheus.CounterOpts{
+		Name: "waku_rln_spam_messages_total",
+		Help: "number of spam messages detected",
```

Comment:
contentTopic would be a good addition, but peerID not so sure. Seems to me that it would be too noisy, and peerIDs do change on restart so not sure if it would add value. pubsubTopic sadly i do not have access to in the validator.

Reply:
Valid point wrt peerID, but i was wondering if it would be useful for service nodes who will be online most of the time. Let's not have peerID for now then. We can add it if needed.
Odd that we don't have access to pubSubTopic in the validator.